### ISSUE TITLE
[question] What about some BOM functionality?

### ISSUE THREAD

**Issue by snelweg** [realthunder/FreeCAD#208]
In Fusion I could export data to for example OpenBOM.
In FreeCAD there is nothing like that, there are some macros but they tend to work with Part cubes and such. There is a document that describes how FC should implement BOM functionality, but it's just talk nothing is working yet.

I can imagine we could give the body or part container properties, in the Arch workbench there are already such editors.\

So basically I am asking if there are plans for making BOM functioning in FC.
